src/commands/PublishCommand.js

```diff
@@ -174,6 +180,11 @@ export default class PublishCommand extends Command {
     this.gitRemote = this.options.gitRemote || "origin";
     this.gitEnabled = !(this.options.canary || this.options.skipGit);
 
+    this.npmConfig = {
+      client: this.options.npmClient || 'npm',
+      registry: this.registry
```

Comment:
I believe this needs to be `registry: this.npmRegistry` (set in [Command.js](https://github.com/lerna/lerna/blob/a20601c6e94b3f651b6a8474dd1b1f00269f6be2/src/Command.js#L314)), as `this.registry` doesn't exist.

Reply:
good catch